shared/src/main/scala/mlscript/Typer.scala

```diff
@@ -442,6 +443,7 @@ class Typer(var dbg: Boolean, var verbose: Bool, var explainErrors: Bool)
         // currently we get things like "flows into variable reference"
         // but we used to get the better "flows into object receiver" or "flows into applied expression"...
       case lit: Lit => ClassTag(lit, lit.baseClasses)(prov)
+      // HERE 
```

Comment:
Here?

Reply:
sorry, I was about to remove it after your review :sweat_smile: